src/components/Card/Button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface ButtonProps {
+   name: string;
+   disabled: any;
+   onClick: any;
```

Comment:
Please try not to use 'any' type.

Reply:
It shouldn't be at your code.

Reply:
I'm understand it, but if i write another type, that get an error from eslint.